applications/utils.py

```diff
@@ -155,8 +156,12 @@ def export_winter_storage_applications_as_xlsx(applications):
                     value = getattr(application, attr_name)
                     if isinstance(value, bool):
                         value = "Yes" if value else ""
-                    elif isinstance(value, Enum):
-                        value = str(value)
+                    elif value in ApplicationStatus:
+                        status = ApplicationStatus(value)
+                        value = str(getattr(status, "label", str(status)))
```

Comment:
application status field is not used when generating excels, you can skip these lines, only `storage_method` change was needed